Library/Homebrew/cask/lib/hbc/installer.rb

```diff
@@ -87,7 +87,7 @@ def summary
     s = if MacOS.version >= :lion && !ENV["HOMEBREW_NO_EMOJI"]
           (ENV["HOMEBREW_INSTALL_BADGE"] || "\xf0\x9f\x8d\xba") + "  "
         else
-          "#{Hbc::Utils::Tty.blue.bold}==>#{Hbc::Utils::Tty.reset.bold} Success!#{Hbc::Utils::Tty.reset} "
+          "#{Tty.blue}==> Success!#{Tty.reset} "
```

Comment:
I believe it's private just because it's not used on its own anywhere, and it's a "composing" implementation method that requires a color argument. Only the actually-used "style" methods are public. If a bold-but-unchanged-color style ends up being needed, I'd see no reason you couldn't rename the existing `Tty.bold` to `Tty.bold_and` and make a public `Tty.bold` method.